Mathlib/RingTheory/Localization/FractionRing.lean

```diff
@@ -226,6 +226,82 @@ noncomputable def fieldEquivOfRingEquiv [Algebra B L] [IsFractionRing B L] (h :
         mem_nonZeroDivisors_iff_ne_zero, mem_nonZeroDivisors_iff_ne_zero]
       exact h.symm.map_ne_zero_iff)
 
+@[simp]
+lemma fieldEquivOfRingEquiv_algebraMap [Algebra B L] [IsFractionRing B L] (h : A ≃+* B)
+    (a : A) : fieldEquivOfRingEquiv h (algebraMap A K a) = algebraMap B L (h a) := by
+  simp [fieldEquivOfRingEquiv]
+
+section fieldEquivOfAlgEquiv
+
+variable {A B C D : Type*}
+  [CommRing A] [CommRing B] [CommRing C] [CommRing D]
+  [IsDomain A] [IsDomain B] [IsDomain C] [IsDomain D]
+  [Algebra A B] [Algebra A C] [Algebra A D]
+  (K L M O : Type*) [Field K] [Field L] [Field M] [Field O]
+  [Algebra A K] [Algebra B L] [Algebra C M] [Algebra D O]
+  [IsFractionRing A K] [IsFractionRing B L] [IsFractionRing C M] [IsFractionRing D O]
```

Comment:
What would your thoughts be on changing this to something like
```suggestion
  [IsFractionRing A Af] [IsFractionRing B Bf] [IsFractionRing C Cf] [IsFractionRing D Df]
```
or
```suggestion
  [IsFractionRing A KA] [IsFractionRing B KB] [IsFractionRing C KC] [IsFractionRing D KD]
```
which makes it a bit easier to keep track of the 8 variables.

Reply:
Sure. I switched to `FA`, `FB`, `FC`, `FD`.